debugview/plugin.cpp

```diff
@@ -145,34 +144,39 @@ class debugview : public plugin {
 			ImGui::Text("Resets to zero'd out tracking universe");
 		}
 		ImGui::Spacing();
-		const pose_type* fast_pose_ptr = pp->get_fast_pose();
+		const pose_type fast_pose_ptr = pp->get_fast_pose();
```

Comment:
Minor: fast_pose_ptr is no longer a `ptr` but the actual fast pose itself. Similar issue on line 149.
```suggestion
		const pose_type fast_pose = pp->get_fast_pose();
```